devserver.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Server for static files.
+ * Usage:
+ *  1. run `yarn devserver:start`
+ *  2. get access to files from `/example` or `/dist` on any device in local network.
+ *     example: `http://{ip_address}:3000/example/example-dev.html`
+ *     where {ip_address} is IP of your machine in local network.
+ */
+const path = require('path');
+const fs = require('fs');
+const http = require('http');
+
+const port = 3000;
+const host = '0.0.0.0';
+const server = http.createServer(serveStatic(['/example', '/dist']));
+
+server.listen(port, host, () => {
+  console.log(`Server is running on http://${host}:${port}`);
```

Comment:
I think we can make this message more developer-friendly. Add some instructions: how to know the network IP, maybe other useful info. And also, the should be a link to the example page.